mitxpro/context_processors.py

```diff
@@ -31,6 +31,9 @@ def configuration_context(request):
         "hubspot_footer_form_guid": settings.HUBSPOT_CONFIG.get(
             "HUBSPOT_FOOTER_FORM_GUID"
         ),
+        "hubspot_enterprise_page_form_id": settings.HUBSPOT_CONFIG.get(
+            "HUBSPOT_ENTERPRISE_PAGE_FORM_ID"
+        ),
```

Comment:
Since `hubspot_footer_form_guid` is coming from the context processor, I suggest adding `hubspot_enterprise_page_form_id` here as well and removing it from models.

Reply:
The reason I am suggesting using models is because context processors will add the ID for every template request where we don't even need it.

Reply:
Got it, thank you.